tools/build_arrow.sh

```diff
@@ -228,6 +214,26 @@ if [ $BUILD_ARROW == "ON" ]; then
   mkdir -p $ARROW_SOURCE_DIR
   mkdir -p $ARROW_ROOT
 
+  if [ -d $ARROW_INSTALL_DIR ]; then
+      rm -r $ARROW_INSTALL_DIR
+  fi
+
+  if [ -d $ARROW_SOURCE_DIR ]; then
+    echo "Applying incremental build for Arrow..."
+    pushd $ARROW_SOURCE_DIR
+    git init .
+    EXISTS=`git show-ref refs/heads/build_$TARGET_BUILD_COMMIT || true`
```

Comment:
Sometimes arrow java incremental build may fail  in my local machine, in case of rat plugin or code change too much

Reply:
Thanks for the information. I'm going to disable the feature by default.